src/views/studio/StudioLayout.tsx

```diff
@@ -47,25 +45,24 @@ const StudioLayout = () => {
   const location = useLocation()
   const displayedLocation = useVideoEditSheetRouting()
   const { isUserConnectedToInternet, nodeConnectionStatus } = useConnectionStatus()
-  const { extensionConnected: extensionStatus, accounts } = useJoystream()
 
   const {
-    activeUser: { accountId, memberId, channelId },
-    loading: activeUserLoading,
-  } = useActiveUser()
-
-  const { memberships, loading: membershipsLoading } = useMemberships({
-    where: { controllerAccount_in: accounts.map((a) => a.id) },
-  })
+    activeAccountId,
+    activeMemberId,
+    activeChannelId,
+    extensionConnected: extensionStatus,
+    accounts,
+    memberships,
+    userInitialized,
+  } = useUser()
+  const extensionConnected = extensionStatus === true
 
   const [enterLocation] = useState(location.pathname)
-  const extensionConnectionLoading = extensionStatus === null
-  const extensionConnected = extensionStatus === true
   const hasMembership = !!memberships?.length
 
-  const accountSet = !!accountId && extensionConnected
-  const memberSet = accountSet && !!memberId && hasMembership
-  const channelSet = memberSet && !!channelId && hasMembership
+  const accountSet = !!activeAccountId && extensionConnected
```

Comment:
```suggestion
  const accountSet = !!activeAccountId && !!extensionConnected
```
sorry, last thing. Should work now

Reply:
Yep, already pushed that :+1: